backend/tests/Feature/SoftDeleteTest.php

```
@@ -0,0 +1,181 @@
+        $this->assertNull(Skill::find($skill->id));
+        $this->assertDatabaseHas('skills', ['id' => $skill->id]);
+
+        $trashed = Skill::withTrashed()->find($skill->id);
+        $this->assertNotNull($trashed->deleted_at);
+        $this->assertSame($this->admin->id, $trashed->deleted_by_id);
+    }
+
+    public function test_delete_cascades_through_the_whole_aggregate(): void
+    {
+        $wo = $this->workOrderAggregate();
+        $batch = $wo->fresh()->id;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$batch'. (undefined)

(UnusedLocalVariable)